charts/security-apps/ci/default-values.yaml

```diff
@@ -38,6 +38,14 @@ neuvectorMonitor:
   enabled: true
   values: {}
 
-oaut2-proxy:
+oauth2-proxy:
```

Comment:
hmm maybe this should go into the release notes or into a bugfix release (except we are pretty sure, that nobody is using this like that)

Reply:
no one is, the ci folder isn't part of the public api and should not be relied upon except for testing.